bootstore/src/schemes/v0/peer.rs

```diff
@@ -0,0 +1,797 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! The entrypoint of the v0 scheme for use by bootstrap agent
+
+use super::peer_networking::{
+    spawn_client, spawn_server, AcceptedConnHandle, ConnToMainMsg,
+    ConnToMainMsgInner, MainToConnMsg, Msg, PeerConnHandle,
+};
+use super::{ApiError, ApiOutput, Config as FsmConfig, Fsm, RackUuid};
+use crate::trust_quorum::RackSecret;
+use derive_more::From;
+use sled_hardware::Baseboard;
+use slog::{debug, error, info, o, warn, Logger};
+use std::collections::{BTreeMap, BTreeSet};
+use std::net::{SocketAddr, SocketAddrV6};
+use std::time::Duration;
+use tokio::net::{TcpListener, TcpStream};
+use tokio::sync::{mpsc, oneshot};
+use tokio::time::{interval, Instant, MissedTickBehavior};
+use uuid::Uuid;
+
+#[derive(Debug, Clone)]
+pub struct Config {
+    id: Baseboard,
+    addr: SocketAddrV6,
+    time_per_tick: Duration,
+    learn_timeout: Duration,
+    rack_init_timeout: Duration,
+    rack_secret_request_timeout: Duration,
+}
+
+// An error response from a `PeerRequest`
```

Comment:
Nit: use docstring comments (`///`) here and above the variants, since some editors use different colors for docstrings vs regular comments.

Reply:
done